docker/entry.py

```diff
@@ -117,6 +117,12 @@ def main():
     except OSError:
         pass
 
+    # Execute any startup executables.
+    for exe in os.listdir('/usr/libexec/pyrex/startup.d'):
+        path = os.path.join('/usr/libexec/pyrex/startup.d', exe)
+        if os.path.isfile(path) and os.access(path, os.X_OK):
+            subprocess.call(path)
```

Comment:
This should probably be:
```python
    try:
        subprocess.check_call([path])
    except subprocess.CalledProcessError as e:
        sys.stderr.write('%s exited with %d\n' % (path, e.returncode))
        sys.exit(e.returncode)
    
```
I think we want a failure in one of the startup scripts to be detected and kill the container startup.

Also note that the script should be passed as a list so that it is invoked directly instead of through the shell interpreter.